.github/workflows/cuda.yml

```diff
@@ -0,0 +1,71 @@
+name: CUDA GitHub Actions
+
+on:
+  pull_request_review_comment:
+    types: [created]
+
+jobs:
+  test:
+    name: CUDA
+    runs-on: [self-hosted, linux]
+    if: github.event.comment.body == '/gha run cuda-builds' && contains('OWNER,MEMBER,COLLABORATOR', github.event.comment.author_association)
```

Comment:
/gha run cuda-builds

Reply:
Comment above triggered the following build: https://github.com/microsoft/LightGBM/runs/1190615093?check_suite_focus=true.

Reply:
/gha run cuda-builds

Reply:
this is so cool!!